app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java

```diff
@@ -72,6 +96,11 @@ public View getChildView(int groupPosition, final int childPosition,
         return convertView;
     }
 
+    /**
+     * returns total children count.
```

Comment:
Thank you @rohands for the clarification. The unnecessary commits are here most probably because @top-gun007 's comments branch is based on PCSA master branch, not the develop branch.